src/utils/PerfherderGraphContainer.jsx

```diff
@@ -60,14 +60,15 @@ const tip = withStyles(tipStyles)(
     const higherOrLower = series.meta.lowerIsBetter
       ? 'lower is better'
       : 'higher is better';
-    const hgURL = URL({
-      path: 'https://hg.mozilla.org/mozilla-central/pushloghtml',
-      query: { changeset: record.revision },
-    });
+
+    const revisionURL = record.meta.repo === 'mozilla-central'
+      ? `https://hg.mozilla.org/mozilla-central/pushloghtml?changeset=${record.revision}`
+      : `https://github.com/mozilla-mobile/fenix/commit/${record.revision}`;
```

Comment:
This should not be text munging.  It would be nice if the URL was yet-another-property, instead of switching in "mozilla-central".

In any case, this is good enough.

Reply:
Thanks @klahnakoski  👍😀